eth2deposit/utils/constants.py

```diff
@@ -13,6 +13,7 @@
 
 
 # File/folder constants
+INTL_CONETENT_PATH = os.path.join('eth2deposit', 'intl')
```

Comment:
"INTL_CONTENT_PATH"?

Reply:
Indeed, tnx. Fixed in https://github.com/ethereum/eth2.0-deposit-cli/pull/182/commits/b8c94a80fc8661db4affb83c13332e8e3369d92e